modules/nw-operator-vsphere-multiple-nic.adoc

```diff
@@ -0,0 +1,63 @@
+// Module included in the following assemblies:
+//
+// * installing/installing_vsphere/installing-vsphere-installer-provisioned-network-customizations.adoc
+// * installing/installing_vsphere/installing-vsphere-network-customizations.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="nw-operator-vsphere-multiple-nic.adoc_{context}"]
+= Adding multiple network interface cards (NICs) to an OVN-Kubernetes network plugin
+
+Before you install an {product-title} cluster on a {vmw-short} host, you can add multiple NICs to an OVN-Kubernetes network plugin configuration that {vmw-full} uses for managing pods and services on your cluster. For this configuration, you must specify internal and external Classless Inter-Domain Routing (CIDR) implementations in the {vmw-short} cloud controller manager (CCM) configuration. Each CIDR lists an IP address range that the CCM uses to determine what NICs interact with traffic from internal and external networks.
```

Comment:
Thanks. I removed the OVNK references. WDYT of:

> Before you install an OCP cluster on a vSphere host, you can specify multiple NICs for a networking implementation so that the vSphere cloud controller manager (CCM) can select the appropriate NIC for a given networking situation. The {vmw-full} can use the NIC for managing pods and services on your cluster. For this configuration, you must specify internal and external Classless Inter-Domain Routing (CIDR) implementations in the vSphere cloud controller manager (CCM) configuration. Each CIDR lists an IP address range that the CCM uses to determine what NICs interact with traffic from internal and external networks.

"networking implementation" OK?